src/store/index.js

```diff
@@ -1,12 +1,13 @@
-import { fillServerWait } from 'mobx-server-wait';
-import PlanetStore from './PlanetStore';
+import { fillServerWait } from 'utils/mobx-server-wait';
+import Network from './Network';
+import Planets from './Planets';
 
 export default class Store {
 
   constructor(state = {}) {
-    const { planets, ...rest } = state;
-    this.planets = new PlanetStore(planets);
-    Object.assign(this, rest);
+    Object.assign(this, state);
+    this.network = new Network(this);
+    this.planets = new Planets(this);
```

Comment:
It's neat that ({ planets, network }) are the arguments to this constructor but this might be more obvious as an explicit Planets(this.planets, this.network) since network is a required parameter.